utils/dvmhost/Makefile

```diff
@@ -0,0 +1,51 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dvmhost
+PKG_VERSION:=1.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=e9abdf63aa656465438d59f009a83faf5ba69f1f	# R04J32 with hotfixes
+PKG_SOURCE_URL:=https://github.com/dvmproject/dvmhost
+PKG_BUILD_PARALLEL:=1
+
+PKG_MAINTAINER:=Natalie Moore <natalie@natalielucy.dev>
+PKG_LICENSE:=GPL-2.0-only
+
+# disable TUI to save space; pending upstream updates for websocket
+CMAKE_OPTIONS+= -DENABLE_TUI_SUPPORT=0 -DDISABLE_WEBSOCKETS=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/dvmhost/Default
```

Comment:
I think a default section is useful when there are multiple subpackages. I only see one so far.

Reply:
There are multiple packages in DVMHost like patch, bridge, FNE, etc - I'm still new and trying to figure out the best way to break these out (whether all separate or one minimal/one full package or something else). I'm open to suggestions! But, I wanted to lay a framework out now to be able to handle the multiple packages later.

Reply:
That's can be done later on, right? :)  I would remove Default section for now.